index.js

```diff
@@ -78,15 +78,28 @@ class S3Grc extends Base {
       cb)
   }
 
-  getDownloadUrl (filename, key, cb = null) {
+  /**
+   * @param {string} filename
+   * @param {string} key
+   * @param {object} [opts]
+   * @param {string} [opts.bucketName]
+   * @param {function} [cb]
```

Comment:
```suggestion
   * @param {object|function|null} [opts]
   * @param {string} [opts.bucketName]
   * @param {function|null} [cb]
```

Reply:
because it is a way to show optionality (you could use `function?`) and if your IDE supports type hints it will show a warning of invalid number of argument because some types are mandatory